src/config.rs

```diff
@@ -136,7 +143,8 @@ impl Arbitrary for SwarmConfig {
             max_elements: u.int_in_range(0..=MAX_MAXIMUM)?,
             max_data_segments: u.int_in_range(0..=MAX_MAXIMUM)?,
             max_instructions: u.int_in_range(0..=MAX_MAXIMUM)?,
-            max_memories: u.int_in_range(0..=(MAX_MAXIMUM as u32))?,
+            max_memories: u.int_in_range(0..=10)?,
```

Comment:
FWIW I reduced this a bit because the limit in wasmparser is smaller than 1000

Reply:
Do you want to make it match `wasmparser`'s max of `100`? Doesn't make sense to constrain generation narrower than `wasmparser` can handle.

Reply:
sgtm